src/posthog-core.ts

```diff
@@ -574,8 +574,8 @@ export class PostHog {
             this.compression = includes(config['supportedCompression'], Compression.GZipJS)
                 ? Compression.GZipJS
                 : includes(config['supportedCompression'], Compression.Base64)
-                  ? Compression.Base64
-                  : undefined
+                ? Compression.Base64
+                : undefined
```

Comment:
I don't know why my local setup keeps tweaking this when committing. I know CI failed before because of this, but I can't fix this because my local hook keeps updating it 😮‍💨

Reply:
Ok, `pnpm install` did it, I had an outdated version of `prettier` in that repo apparently